cardano-config/cardano-config.cabal

```diff
@@ -31,6 +33,7 @@ library
                      , cardano-shell
                      , cborg
                      , contra-tracer
+                     , file-embed
```

Comment:
why is this new dependency needed?

Reply:
Because git revision fetching was moved from `cardano-node` to `cardano-config`.